src/service/api/offer.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const {Router} = require(`express`);
+
+const {HttpCode} = require(`../../constants`);
+const offerValidator = require(`../middlewares/offer-validator`);
+const offerExist = require(`../middlewares/offer-exists`);
+const commentValidator = require(`../middlewares/comment-validator`);
+const commentExist = require(`../middlewares/comment-exist`);
+
+
+const route = new Router();
+
+module.exports = (app, offerService, commentService) => {
+  app.use(`/offers`, route);
+
+  route.get(`/`, async (req, res) => {
+    const offers = await offerService.findAll();
+    res.status(HttpCode.OK)
+      .json(offers);
```

Comment:
есть смысл разделять определения от логики (на подобие как сделано ниже)
```suggestion
    const offers = await offerService.findAll();
    
    res.status(HttpCode.OK)
      .json(offers);
```